lib/views/pages/on-site.html

```diff
@@ -1,3 +1,40 @@
 <% extend 'lib/views/layouts/default' %>
 
 <h2>On Site</h2>
+<p>Welcome to our event. We appreciate that you attend this year’s NightlyBuild edition and hope you enjoy the evening.<br>
+To make your experience as comfortable as possible, you can find the most important things on this page.</p>
+
+<div class="theme">
+    <blockquote>
+        tl;dr: Don't be a jerk.
+    </blockquote>
+</div>
+
+<h3>Code of Conduct</h3>
+<p>
+    As an inclusive event,
```

Comment:
Sorry, to give you a bit more context (was a bit late yesterday already) on what I mean here: I was following the concept of [semantic linebreaks](http://blog.doismellburning.co.uk/semantic-linebreaks/) here, but realize that I did only for this section though…